habitat/hooks/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cd /hab/svc/dcob
+
+export RACK_ENV=production
+export SSL_CERT_FILE="$(hab pkg path core/cacerts)/ssl/certs/cacert.pem"
```

Comment:
Makes me weepy to do this now. I suppose if there is a way to write out our own run hook in some phase of the plan, we could generate this at build time.